src/main/java/com/github/black0nion/blackonionbot/commands/information/GuildInfoCommand.java

```diff
@@ -1,35 +1,34 @@
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackUser;
-import com.github.black0nion.blackonionbot.bot.BotInformation;
-import com.github.black0nion.blackonionbot.commands.TextCommand;
-import com.github.black0nion.blackonionbot.commands.CommandEvent;
-import net.dv8tion.jda.api.entities.Message;
 import net.dv8tion.jda.api.entities.TextChannel;
-import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
-
-import java.time.format.DateTimeFormatter;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import org.jetbrains.annotations.NotNull;
 
-public class GuildInfoCommand extends TextCommand {
+public class GuildInfoCommand extends SlashCommand {
 
 	public GuildInfoCommand() {
-		this.setCommand("guildinfo", "serverinfo");
+		super(builder(Commands.slash("guildinfo", "Provides information about the current guild.")));
 	}
 
 	@Override
-	public void execute(final String[] args, final CommandEvent cmde, final MessageReceivedEvent e, final Message message, final BlackMember member, final BlackUser author, final BlackGuild guild, final TextChannel channel) {
+	public void execute(@NotNull SlashCommandEvent cmde, SlashCommandInteractionEvent e, BlackMember member, BlackUser author, @NotNull BlackGuild guild, TextChannel channel) {
 		cmde.reply(cmde.success().setTitle("guildinfo")
-			.setThumbnail(guild.getIconUrl())
-			.addField("name", guild.getEscapedName(), true)
-			.addField("language", guild.getLanguage() != null ? (guild.getLanguage().getName() + " (" + guild.getLanguage().getLanguageCode() + ")") : "none", true)
-			.addField("owner", guild.retrieveOwner().submit().join().getUser().getAsMention(), true)
-			.addField("serverid", guild.getId(), true)
-			.addField("rolecount", String.valueOf(guild.getRoles().size()), true)
-			.addField("membercount", String.valueOf(guild.getMemberCount()), true)
-			.addField("channelcount", String.valueOf(guild.getChannels().size()), true)
-			.addField("boostlevel", guild.getBoostTier().name(), true)
-			.addField("created", guild.getTimeCreated().format(BotInformation.DATE_TIME_FORMATTER), true));
+				.setThumbnail(guild.getIconUrl())
+				.addField("name", guild.getEscapedName(), true)
+				.addField("language", guild.getLanguage() != null ? (guild.getLanguage().getName() + " (" + guild.getLanguage().getLanguageCode() + ")") : "none", true)
+				.addField("owner", guild.retrieveOwner().submit().join().getUser().getAsMention(), true)
+				.addField("serverid", guild.getId(), true)
+				.addField("rolecount", String.valueOf(guild.getRoles().size()), true)
+				.addField("membercount", String.valueOf(guild.getMemberCount()), true)
+				.addField("channelcount", String.valueOf(guild.getChannels().size()), true)
+				.addField("boostlevel", guild.getBoostTier().name(), true)
+				.addField("created", BotInformation.DATE_PATTERN.format(guild.getTimeCreated()), true));
```

Comment:
I recently changed the way to format the date; please use `guild.getTimeCreated().format(BotInformation.DATE_TIME_FORMATTER)`